node-notes-cli/app.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+
+const arg1 = process.argv[2];
+
+if (arg1 === 'read') {
+  fs.readFile('./data.json', 'utf8', (err, data) => {
+    if (err) {
+      throw err;
+    }
+    console.log(data);
+  });
+} else if (arg1 === 'create') {
+  fs.readFile('./data.json', 'utf8', (err, data) => {
+    if (err) {
+      throw err;
+    }
+    const arg2 = process.argv[3];
+
+    const newJSON = JSON.parse(data);
+    newJSON.notes[newJSON.nextId.toString()] = arg2;
```

Comment:
You don't have to use `.toString` for the property name, it will work the same if it is a number or a string

Reply:
gotcha